src/sentry/api/urls.py

```diff
@@ -3196,6 +3191,12 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         PromptsActivityEndpoint.as_view(),
         name="sentry-api-0-prompts-activity",
     ),
+    # Feature Flag Providers
+    re_path(
+        r"^flags/hooks/provider/(?P<provider>[\w-]+)/token/(?P<token>.+)/$",
```

Comment:
With this URL, you will not be able to send requests to the `sentry.io` domain. Your endpoint will only work with `us.sentry.io` or `de.sentry.io`. You would need an organization slug to go through the control silo proxies.

Reply:
Ah okay.  I'll change the URL thanks mark.